### PR TITLE
Newsletter 13: Add voxel-mapper

### PR DESCRIPTION
**Pull request by bonsairobo**
https://github.com/rust-gamedev/rust-gamedev.github.io/issues/232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -66,6 +66,23 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Library & Tooling Updates
 
+### voxel-mapper
+
+![voxel-mapper-img](https://raw.githubusercontent.com/amethyst/voxel-mapper/master/screenshots/blending.png)
+
+[voxel-mapper](https://github.com/amethyst/voxel-mapper) is a new library and
+in-game editor for voxel maps, smooth or cubey. The ultimate goal of the project
+is to make it easy for artists and programmers alike to generate volumetric game
+content, either manually or procedurally. The library's author @bonsairobo has
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:76]: Might be nice to add a link to the author here, if they've got a Twitter/blog/etc?

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good :+1:

**Review by ozkriff — APPROVED** [review]
:+1: (Embedd the image and made a few style tweaks)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**